### PR TITLE
Easy for code reading

### PR DESCRIPTION
**Pull request by wangkeqiang123**
add blank and modify some words.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2042)

### CODE REVIEW

File: testing/vm_util.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def wait_for_operation(client,
     op: The final operation.
 
   Raises:
-    TimeoutError: if we timeout waiting for the operation to complete.
+    TimeoutError: Waiting for operation to complete timeout.
```
Comment (lluunn) [inline testing/vm_util.py]: I think the original one is more clear

File: testing/test_deploy.py [hunk 2]
```diff
@@ -446,7 +446,7 @@ def deploy_minikube(args):
       raise
     content = json.loads(e.content)
     if content.get("error", {}).get("code") == requests.codes.CONFLICT:
-      # We don't want to keep going so we reraise the error after logging
+      # We don't want to keep going so we raise the error after logging
```
Comment (lluunn) [inline testing/test_deploy.py]: this is intended to be reraise (or re-raise), right?

File: testing/vm_util.py (hunk 1 above)
Comment (wangkeqiang123) [reply testing/vm_util.py]: Okay

File: testing/test_deploy.py (hunk 2 above)
Comment (wangkeqiang123) [reply testing/test_deploy.py]: 'reraise' may be the better choice.

Comment (wangkeqiang123) [reply testing/test_deploy.py]: 'reraise' may be the better choice.

**Comment by wangkeqiang123** [comment]
@lluunn @jlewi  Pls review my PR in your spare time, thanks a lot.

**Comment by wangkeqiang123** [comment]
@lluunn @jlewi pls leaves an /ok-to-test message. Thans very much.

**Pull request merged**